CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contribute to this project
+
+This page contains pointers and links to help you contribute to this project.
+
+## Our team compass
+
+The {external+tc:doc}`Team Compass <index>` is a source of truth for our team structure and policy.
+It also has a lot of information about how to contribute.
+
+## Code of conduct
+
+We expect all contributors to this project to [](inv:tc#code-of-conduct).
```

Comment:
In order to better display these links, I suggest to use reference-style link, i.e.
```markdown
[Text to display][link-id]

[Link-id]: /path/to/object
```
Combined with `include` with `start-after` `end-before`, you can use html comments like `<!-- MYST-START -->` to separate the text part from the actual link and have `inv:` style links in the html document and actual links in the Github rendered view.

Example: https://github.com/LecrisUT/CMake-Template/blob/main/README.md?plain=1

Reply:
This is a good idea @LecrisUT. We have an [outstanding issue](https://github.com/executablebooks/mystmd/issues/851) about implementing the link scheme resolution for intersphinx, which right now changes between MySTMD and myst-parser